### PR TITLE
Make elb_name a configurable param

### PR DESCRIPTION
**Pull request by achanda**

### CODE REVIEW

File: vars.tf [hunk 1]
```diff
@@ -63,6 +63,10 @@ variable availibility_zones {
   type = "list"
 }
 
+variable elb_name {
+  type = "string"
```
Comment (tomwilkie) [inline vars.tf:67]: Could you add the default (`kubernetes-master`) to this please?

**Comment by tomwilkie** [comment]
One comment, otherwise looks good - thanks!

File: vars.tf (hunk 1 above)
Comment (achanda) [reply vars.tf:67]: Done

Comment (tomwilkie) [reply vars.tf:67]: thanks!

**Comment by tomwilkie** [comment]
Thanks!

**Pull request merged**